<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsGeneralTermsInfoAdobe</class>
 <widget class="QWidget" name="SettingsGeneralTermsInfoAdobe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>1061</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>849</width>
    <height>1061</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true" />
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="settingsSectionHeader" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="SilentQPushButton" name="backBtn">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Atr&#225;s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sectionTitleLbl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>14142</height>
         </size>
        </property>
        <property name="text">
         <string>Licencia Adobe</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>57</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="settingsMarginTop">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="settingsMenuListCont" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true" />
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="bqQTextBrowser" name="generalTermsResult">
        <property name="minimumSize">
         <size>
          <width>562</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>795</width>
          <height>1414214</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd"&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name="qrichtext" content="1" /&gt;&lt;style type="text/css"&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=" font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;"&gt;
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;a name="SEC1"&gt;&lt;/a&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; font-weight:600; color:#000000;"&gt;G&lt;/span&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; font-weight:600; color:#000000;"&gt;ADOBE LICENSE&lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Courier New,courier'; font-size:9pt; color:#000000;"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. &lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Courier New,courier'; font-size:9pt; color:#000000;"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/span&gt;&lt;/p&gt;
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier'; font-size:9pt; color:#000000;"&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Courier New,courier'; font-size:9pt; color:#000000;"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit.&lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Courier New,courier'; font-size:9pt; color:#000000;"&gt;Nullam adipiscing fringilla lacus, in laoreet nibh cursus ac.&lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;a name="preamble"&gt;&lt;/a&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; font-weight:600; color:#000000;"&gt;P&lt;/span&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; font-weight:600; color:#000000;"&gt;reamble&lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam adipiscing fringilla lacus, in laoreet nibh cursus ac. Vestibulum suscipit, odio vel tempor sollicitudin, sapien mi malesuada urna, nec porta lorem leo nec eros. Vivamus tristique porta orci a faucibus. Nam vitae nunc erat. Quisque euismod turpis in nisl mollis viverra. In elementum luctus enim sit amet porta. Duis mi quam, interdum at lacinia ut, aliquet id arcu. Donec tincidunt nulla id odio porta in adipiscing nisl posuere. Donec sed eros nunc, et semper nibh. Suspendisse condimentum, ante in ullamcorper auctor, risus odio accumsan tellus, at tempus libero elit et odio. &lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;Proin suscipit, ipsum vitae viverra fringilla, purus ante placerat elit, quis vestibulum urna erat in purus. Suspendisse egestas molestie turpis, at interdum tellus scelerisque sit amet. Donec nec turpis sed nisi aliquet ultrices. Donec feugiat nunc eget libero condimentum sit amet ullamcorper enim vulputate. Sed gravida blandit sem at faucibus. Curabitur fermentum tortor est. Nunc quis tellus a nibh cursus pharetra. Morbi non ullamcorper turpis. Nunc a eros dapibus diam lobortis aliquam eu quis ligula. In eu leo sapien, vitae malesuada massa. &lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;Fusce ante magna, imperdiet commodo ultricies a, dictum ut elit. Maecenas pretium pretium purus tristique hendrerit. Donec et vulputate purus. Integer lorem odio, euismod eget egestas non, pretium molestie ipsum. Phasellus aliquam, lectus fermentum laoreet dictum, arcu massa pellentesque arcu, a lobortis mi velit nec urna. Pellentesque rutrum ultricies eros at eleifend. Nam id lectus ac ante condimentum rutrum non vitae arcu. &lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;Morbi aliquam, ipsum sed luctus rutrum, orci erat varius mi, ut laoreet mauris ipsum eu ipsum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In vulputate, tellus vitae sagittis dapibus, dui felis blandit turpis, at sagittis nunc orci sollicitudin libero. Praesent et dapibus nulla. Vestibulum sodales nibh facilisis risus congue sed euismod nunc pulvinar. Nunc nec ipsum venenatis quam commodo interdum pellentesque ut leo. Suspendisse hendrerit mattis urna quis ornare. Cras eleifend, magna scelerisque pharetra consectetur, eros ligula placerat urna, vel gravida tellus mi blandit dolor. &lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;We protect your rights with two steps: (1) copyright the software, and (2) offer you this license which gives you legal permission to copy, distribute and/or modify the software.&lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;Cras vestibulum tellus at arcu lobortis lacinia. Maecenas erat nulla, lacinia ut ullamcorper id, dapibus id dui. Pellentesque sapien dolor, tristique a faucibus eget, tincidunt in nibh. Mauris interdum feugiat pellentesque. Pellentesque semper laoreet neque sed venenatis. Integer vel felis commodo mauris vehicula imperdiet euismod at tortor. Morbi urna velit, dapibus sed egestas quis, elementum sodales magna. Integer eget nisi ac quam vehicula gravida. Quisque nec lacus neque, aliquam scelerisque est. Sed commodo varius tristique. Nulla facilisi. Nulla accumsan semper tincidunt. &lt;/span&gt;&lt;/p&gt;
&lt;p style=" margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;&lt;span style=" font-family:'Times New Roman'; font-size:9pt; color:#000000;"&gt;Mauris mi nisl, luctus vitae dignissim a, tempor sed arcu. Praesent vestibulum consequat elit, sit amet tincidunt magna gravida vel. Morbi neque orci, sagittis ac suscipit interdum, molestie a libero. Morbi scelerisque, orci eleifend lobortis vulputate, libero erat sagittis orci, eget venenatis lorem mauris quis eros. Curabitur consectetur eleifend lobortis. Praesent elementum mauris non ligula lacinia sodales. Suspendisse aliquam porta volutpat. Nulla id ornare quam. Nunc ultricies bibendum ipsum, vitae fermentum purus bibendum nec. Praesent eget dui ac augue accumsan lobortis. Suspendisse in tortor eu dui dignissim sollicitudin sodales ac mi. Suspendisse potenti. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Fusce non magna nisl. &lt;/span&gt;&lt;/p&gt;
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier New,courier'; font-size:9pt; color:#000000;"&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;"&gt;\&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SettingsPagerCont" name="verticalPage" native="true" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalBottomSection">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>28</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SilentQPushButton</class>
   <extends>QPushButton</extends>
   <header>SilentQPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>bqQTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header location="global">bqQTextBrowser.h</header>
  </customwidget>
  <customwidget>
   <class>SettingsPagerCont</class>
   <extends>QWidget</extends>
   <header>SettingsPagerCont.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources />
 <connections />
</ui>